fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java

```diff
@@ -205,6 +210,23 @@ public Map<String, ColumnStatistic> getTotalColumnStatisticMap() {
         return totalColumnStatisticMap;
     }
 
+    /**
+     * disable join reorder if any table row count is not available.
+     */
+    public static void disableJoinReorderIfTableRowCountNotAvailable(
+            List<LogicalOlapScan> scans, CascadesContext context) {
+        StatsCalculator calculator = new StatsCalculator(context);
+        for (LogicalOlapScan scan : scans) {
+            double rowCount = calculator.getOlapTableRowCount(scan);
+            if (rowCount == -1 && ConnectContext.get() != null) {
```

Comment:
-1 is a magic number could we add a static variable `UNKNOWN_ROW_COUNT` to static utils and replace all magic number `-1` by the static variable

Reply:
this is the return value of OlapTable.getRowCountForIndex().
explained the "-1" at the spec of OlapTable.getRowCountForIndex()